### PR TITLE
chore: fix the limit plugins schema

### LINKED ISSUE #3453: bug: limit-xxx plugins schema bug
### Issue description

limit-count:

When the `count` or `time_window` fields of the limit-count plugin are 0, the plugin is not available.

Repair method:
It should be greater than 0 in the schema. Need to provide response test cases.

imit-req:

When the `rate`  fields of the limit-req plugin are 0, the plugin is not available.

Repair method:
It should be greater than 0 in the schema. Need to provide response test cases.

### Environment

* apisix version (cmd: `apisix version`): 2.2
* OS (cmd: `uname -a`):  Liunx
* OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`): nginx version: openresty/1.19.3.1
* etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
* apisix-dashboard version, if have:

### Minimal test code / Steps to reproduce the issue

1. add route

```shell
curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
{
    "uri": "/hello",
    "plugins": {
        "limit-count": {
            "count": 0,
            "time_window": 60,
            "rejected_code": 503,
            "key": "remote_addr"
        }
    },
    "upstream": {
        "type": "roundrobin",
        "nodes": {
            "127.0.0.1:1980": 1
        }
    }
}'
```
2. test plugin:

```shell
curl -i http://127.0.0.1:9080/hello
HTTP/1.1 500 Internal Server Error
Date: Fri, 29 Jan 2021 04:14:01 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 174
Connection: close
Server: APISIX/2.2

<html>
<head><title>500 Internal Server Error</title></head>
<body>
<center><h1>500 Internal Server Error</h1></center>
<hr><center>openresty</center>
</body>
</html>

```

`/usr/local/openresty/lualib/resty/limit/count.lua` code segment:

```lua
......
assert(limit > 0 and window > 0)
......
```

`/usr/local/openresty/lualib/resty/limit/req.lua` code segment:

```lua
......
assert(rate > 0 and burst >= 0)
......
```

### PR DESCRIPTION
**Pull request by Firstsawyou**
### What this PR does / why we need it:

fix #3453

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Comment by idbeta** [comment]
you should update the doc.

**Review by tokers — APPROVED** [review]

**Review by spacewander — CHANGES_REQUESTED** [review]
Yes, @idbeta is right. We need to update the doc:
https://github.com/apache/apisix/blob/master/doc/plugins/limit-count.md#attributes

**Comment by Firstsawyou** [comment]
> you should update the doc.

updated.

**Comment by Firstsawyou** [comment]
> Yes, @idbeta is right. We need to update the doc:
> https://github.com/apache/apisix/blob/master/doc/plugins/limit-count.md#attributes

updated.

File: doc/plugins/limit-conn.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ Limiting request concurrency plugin.
...
-| rejected_code      | string  | optional    | 503     | [200,...]                                                                | returned when the request exceeds `conn` + `burst` will be rejected.                                                                                                                                                                                                                                                                                                                                                                                            |
+| rejected_code      | string  | optional    | 503     | [200,...,600]                                                                | returned when the request exceeds `conn` + `burst` will be rejected.                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (spacewander) [inline doc/plugins/limit-conn.md:42]: The max of rejected_code should be 599. We should also update the code.

Comment (Firstsawyou) [reply doc/plugins/limit-conn.md]: updated.

File: doc/plugins/limit-conn.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Limiting request concurrency plugin.
...
-| rejected_code      | string  | optional    | 503     | [200,...,600]                                                                | returned when the request exceeds `conn` + `burst` will be rejected.                                                                                                                                                                                                                                                                                                                                                                                            |
+| rejected_code      | string  | optional    | 503     | [200,...,599]                                                                | returned when the request exceeds `conn` + `burst` will be rejected.                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (spacewander) [inline doc/plugins/limit-conn.md:42]: Better also update the schema.

Comment (Firstsawyou) [reply doc/plugins/limit-conn.md:42]: updated.

**Review by spacewander — APPROVED** [review]

**Pull request merged**